python/chatting_sample.py

```diff
@@ -0,0 +1,390 @@
+                    return False
+                if error_type == "MissingBreak":
+                    if break_error_info["Confidence"] >= threshold and last_word.get("is_punctuation", False):
+                        return True
+                if error_type == "UnexpectedBreak":
+                    if break_error_info["Confidence"] >= threshold and not last_word.get("is_punctuation", False):
+                        return True
+                return False
+            elif error_type == "Monotone" and \
+                    error_type in word["PronunciationAssessment"]["Feedback"]["Prosody"]["Intonation"]["ErrorTypes"]:
+                return True
+            else:
+                return False
+
+        def set_error_dict(json_words):
+            for idx, word in enumerate(json_words):
+                if get_prosody_error("MissingBreak", word, json_words[idx-1]):
+                    error_dict["Missing break"].append(word)
+                elif get_prosody_error("UnexpectedBreak", word, json_words[idx-1]):
+                    error_dict["Unexpected break"].append(word)
+                elif get_prosody_error("Monotone", word, json_words[idx-1]):
+                    error_dict["Monotone"].append(word)
+
+        def get_error_message(error_types):
+            message = ""
+            for error_type in error_types:
+                if len(error_dict[error_type]) != 0:
+                    message += f"{error_type} count is {len(error_dict[error_type])}. near the word "
+                    message += f"{' '.join([word['Word'].strip() for word in error_dict[error_type]])}. "
+
+            return message
+
+        def get_report(json_words, mis_pronunciation_words):
+
+            set_error_dict(json_words)
+
+            origin_content = ""
+            if len(mis_pronunciation_words) != 0:
+                origin_content = "Accuracy report:"
+                for mis_word in mis_pronunciation_words:
+                    phone = "".join([syllable.syllable for syllable in mis_word.syllables])
+                    origin_content += f" word {mis_word.word}"
+                    origin_content += f" correct pronunciation is {mis_word.word},"
+                    origin_content += (
+                        f' your pronunciation is <phoneme alphabet="ipa" ph="{phone}">'
+                        f'{mis_word.word}'
+                        '</phoneme>. '
+                    )
+
+            if scores_dict["fluency score"] < 60 or scores_dict["prosody score"] < 60:
+                if scores_dict["fluency score"] < 60:
+                    origin_content += "Fluency "
+                if scores_dict["prosody score"] < 60:
+                    origin_content += "Prosody "
+                origin_content += "report: "
+                origin_content += get_error_message(["Missing break", "Unexpected break", "Monotone"])
+
+            tts(f'<speak version="1.0" xmlns="http://www.w3.org/2001/10/synthesis" xml:lang="en-US">\
```

Comment:
Please check this output. I run this sample but the output wave is illegal and I can not play it.